src/test/java/net/datafaker/FakerTest.java

```diff
@@ -162,7 +162,9 @@ void expression() {
         assertThat(faker.expression("#{letterify '????','true'}")).matches("[A-Z]{4}");
         assertThat(faker.expression("#{templatify '????','?','1','2','q','r'}")).matches("([12qr]){4}");
         assertThat(faker.expression("#{Name.first_name} #{Name.first_name} #{Name.last_name}")).matches("[a-zA-Z']+ [a-zA-Z']+ [a-zA-Z']+");
-        assertThat(faker.expression("#{number.number_between '1','10'}")).matches("[1-9](\\.[0-9]+)?");
+        assertThat(faker.expression("#{number.number_between '1','10'}")).matches("[1-9]([0-9]+)?");
```

Comment:
I think this regex can be simplified: `[1-9]\\d*`  @snuyanzin

Reply:
thanks, done